gradle.properties

```diff
@@ -15,3 +18,6 @@ developers=Graeme Rocher
 
 githubCoreBranch=3.0.x
 bomProperty=micronautCassandraVersion
+
+org.gradle.caching=true
+org.gradle.parallel=true
```

Comment:
Still the [outstanding question](https://github.com/micronaut-projects/micronaut-redis/pull/257#pullrequestreview-878610515) about this before merging